.renovaterc.json

```diff
@@ -5,57 +5,60 @@
     "group:recommended",
     "helpers:disableTypesNodeMajor"
   ],
-
   "ignorePaths": [
     "**/node_modules/**",
     "**/__tests__/**",
     "**/test/**"
   ],
-
   "automerge": false,
   "branchPrefix": "renovate/",
   "ignoreUnstable": true,
   "rangeStrategy": "bump",
   "statusCheckVerify": true,
   "updateNotScheduled": true,
-
   "lockFileMaintenance": {
     "enabled": true,
     "schedule": "before 5am on monday"
   },
-
   "prConcurrentLimit": 5,
   "prCreation": "immediate",
   "prHourlyLimit": 2,
-
   "semanticCommits": true,
   "semanticCommitType": "chore",
   "semanticCommitScope": null,
-
   "separateMajorMinor": true,
   "separateMinorPatch": false,
-
-  "packageRules": [{
-    "packageNames": [
-      "@microsoft/api-extractor",
-      "@microsoft/api-documenter"
-    ],
-    "groupName": "api-extractor packages"
-  },
-  {
-    "packageNames": [
-      "@types/express",
-      "@types/express-serve-static-core"
-    ],
-    "groupName": "@types/express* packages"
-  }
+  "packageRules": [
+    {
+      "packageNames": [
+        "@microsoft/api-extractor",
+        "@microsoft/api-documenter"
+      ],
+      "groupName": "api-extractor packages"
+    },
+    {
+      "packageNames": [
+        "@types/express",
+        "@types/express-serve-static-core"
+      ],
+      "groupName": "@types/express* packages"
+    },
+    {
+      "updateTypes": ["minor", "patch"],
+      "automerge": true
```

Comment:
didn't see where we're defining CI needs to be green in order to automerge. Did I miss something?

Reply:
That's built-in with Renovate, which won't merge any PR if CI not green.